lib/tc-babel.js

```diff
@@ -0,0 +1,123 @@
+"use strict";
+var babel = require('babel-core');
+var fs = require('fs');
+var path = require('path');
+var assert = require('assert');
+var _ = require('lodash');
+var walk = require('fs-walk');
+var mkdirP = require('mkdirp');
+
+/**
+ * This is a library for compiling es6 to node-0.12+ compatible JS.  We wrote
+ * this because there are a couple of bugs in the CLI wrapper for babel.  We
+ * also wanted to implement a system for sharing babel rules between components
+ * through a module.  We do not use es6 to write this script to keep things
+ * simple by avoiding any weird bootstrapping issues with Babel
+ */
+
+// Prior art, a grunt task for doing babel transpiling: 
+//   https://github.com/babel/grunt-babel/blob/master/tasks/babel.js
+
+function deleteSync(dir) {
+  if (fs.lstatSync(dir).isDirectory()) {
+    if (fs.existsSync(dir)) {
+      fs.readdirSync(dir).forEach(function(dirMember) {
+        var fullPath = path.join(dir, dirMember);
+        if (fs.lstatSync(fullPath).isDirectory()) {
+          deleteSync(fullPath);
+        } else {
+          fs.unlinkSync(fullPath);
+        }
+      });
+      fs.rmdirSync(dir);
+    }
+  } else {
+    fs.unlinkSync(dir);
+  }
+}
```

Comment:
nit, but there is also `rmrf` on npm :)

(probably I should find a list folder function on npm)